yt/yt/core/net/dialer.cpp

```diff
@@ -359,6 +368,16 @@ class TAsyncDialerSession
             Timeout_ *= Config_->RtoScale * GetRandomVariation();
         }
 
+        if (Deadline_ && TInstant::Now() >= *Deadline_) {
+            auto error = TError(NRpc::EErrorCode::TransportError, "Connect timed out")
+                << TErrorAttribute("Timeout", Config_->ConnectTimeout);
```

Comment:
Timeout -> timeout (attributes are in lower case)

Reply:
"Connect timed out" above vs "Connect timeout" below -- let's choose a single spelling